web/package.json

```diff
@@ -131,7 +132,8 @@
     "prettier": "^2.0.1",
     "pro-download": "1.0.1",
     "puppeteer-core": "^4.0.1",
-    "stylelint": "^13.0.0"
+    "stylelint": "^13.0.0",
+    "webpack": "^5.35.0"
```

Comment:
Do we need to maintain webpack manually here?

Reply:
yes, need to include webpack here.

Reply:
Ant Design Pro contains UmiJS, and it will handle Webpack configuration.

Reply:
I find it is a [bug](https://github.com/umijs/umi/issues/6398),I will update dependency version to fix it.

Reply:
Is there a convenient way to divide PR?

Reply:
No, I see this PR contains more than 3k+ changes 😳😳

Reply:
Most of the changes are change of yarn.lock . the actual number of lines changed is about 800.